random_name.js

```diff
@@ -0,0 +1,31 @@
+/* Develop function that will handle random name generations.
+
+The function will return a randomly generated name, that is defined as ```description``` + ```food```, similar to the feature found on Google Docs.
+
+Examples
+```js
+generateRandomName () // Fried Cake
+generateRandomName () // Soggy Drumstick
+generateRandomName () // Crispy Noodle
+```
+
+Ideally, it should be designed in a way where we can easily change the available keywords
+*/
+
+function generateRandomName() {
+
+    const adjectives = ['Fried', 'Soggy', 'Crispy'];
+    const meals = ['Cake', 'Drumstick', 'Noodle'];
+
+    var adj = adjectives[Math.floor(Math.random() * (adjectives.length - 0))];
```

Comment:
Typically, you don't ever use ```var``` anymore in modern tech, it was the standard for ES5 javascript. Nowadays (ES6), we use ```const``` and ```let``` exclusively.

```js
(adjectives.length - 0))]
```
No need to subtract 0

Reply:
done